index.html

```diff
@@ -2,15 +2,99 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+<header class="header">
+    <img class="karam-logo" src="img/karma-logo.svg" alt="picture of karam logo">
+    <nav class="nav">
+        <ul class="nav-list">
 
+         <li class="nav-item">
+         <a class="nav-link" href="#">Meet Karma</a>  </li>
+
+         <li class="nav-item">
+            <a class="nav-link" href="#">How it Works </a> </li>   
+
+         <li class="nav-item">
+            <a class="nav-link" href="store.html">Store</a> </li>   
+
+         <li class="nav-item"> 
+            <a class="nav-link" href="#">Blog</a> </li>  
+         
+         <li class="nav-item">
+            <a class="nav-link" href="#">Help</a> </li> 
+            
+         <li class="nav-item">
+            <a class="nav-link" href="#">Login</a> </li>     
+        </ul>
+    </nav>
+   
+</header>
+ 
+<main>
+ <div class="hero">
+    <div> 
+    <h1>introducing Karma</h1>
+    <p>Bring WiFi with you, everywhere you go.</p>
+    <button class="hero-button">Learn More</button>
+ 
+ </div>
+ </div>
+</main>
+
+<section class="section">
+<p class="section-p">Everyone needs a little Karma.</p> 
+
+<div class="section-icon">
```

Comment:
this section is position to the left of the screen, It should be positioned at the center. Instead of adding a `padding-left: 7em` you can remove that and use a `justify-content: center;` for a better result.